sdk/Sdk/Sdk.csproj

```diff
@@ -1,8 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <MinorProductVersion>17</MinorProductVersion>
-    <PatchProductVersion>4</PatchProductVersion>
+    <MinorProductVersion>18</MinorProductVersion>
+    <PatchProductVersion>0</PatchProductVersion>
+    <VersionSuffix>-preview1</VersionSuffix>
```

Comment:
Is an SDK update needed with this set of changes, yet? No TFM or dependency changes, right?

Reply:
Nevermind - I saw the note in the PR description. Could you please provide a link to that changeset, though?

Reply:
Added in PR description.